src/store/channels/saga.ts

```diff
@@ -61,6 +62,8 @@ export function* markConversationAsRead(conversationId) {
   const conversationInfo = yield select(rawChannelSelector(conversationId));
   if (conversationInfo?.unreadCount > 0) {
     yield call(markAllMessagesAsRead, conversationId, currentUser.id);
+    const chatClient = yield call(chat.get);
+    yield call([chatClient, chatClient.markRoomAsRead], conversationId);
```

Comment:
Wouldn't the intent be that this call replaces the one just above? Why would we have both?

Reply:
I've moved this to a more appropriate place. `markChannelsAsRead` and `markConversationAsRead` haven't needed any updates, instead I have updated `markAllMessagesAsRead` to handle/use the chatClient